src/commands/duel.ts

```diff
@@ -156,9 +165,17 @@ class Duel {
     })
   }
 
+  plural(streak: number, outcome: 'win' | 'loss'): string {
+    if (outcome === 'win') {
+      return streak === 1 ? 'win' : 'wins'
+    } else {
+      return streak === 1 ? 'loss' : 'losses'
+    }
+  }
+
   @Slash('duelstats', { description: 'Display your duel statistics' })
   private async duelStats(interaction: CommandInteraction) {
-    await interaction.deferReply()
+    await interaction.deferReply({ ephemeral: true })
```

Comment:
We haven't decided whether the command should only be visible to yourself or to everyone.

Reply:
I think it's more fun to see others streaks. Lets have it show for now, even if it spams chat for a few messages as others check.